web/src/beta/components/StorytellingPageSectionItem/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import StorytellingPageSectionItem from ".";
+
+export default {
+  component: StorytellingPageSectionItem,
+} as Meta;
+
+type Story = StoryObj<typeof StorytellingPageSectionItem>;
+
+export const Default: Story = {
+  render: () => (
+    <StorytellingPageSectionItem icon={"square"} title={"New Page"} index={1} active={false} />
+  ),
```

Comment:
```suggestion
  args: {
    icon: "square",
    title: "New Page",
    index: 1,
    active: false,
  },
```

You can use `args` instead of `render` if you need merely to change the props.
If you use args, storybook attaches `controls` automatically.

![image](https://github.com/reearth/reearth/assets/16318727/03b564d9-3e0d-400f-831e-26121198dbb2)

Reply:
Sorry @atnagata for the confusion. After further looking at storybook docs I think the way @isoppp suggests is better.
If we use args, as @isoppp said we get controls automatically, but we also don't need to write the render!